packages/ember-metal/lib/utils.js

```diff
@@ -676,7 +676,7 @@ var needsFinallyFix = (function() {
 
 var tryFinally;
 if (needsFinallyFix) {
-  tryFinally = function(tryable, finalizer, binding) {
+  tryFinally = (tryable, finalizer, binding) => {
     var result, finalResult, finalError;
 
     binding = binding || this;
```

Comment:
Again, we're changing the context for `this` here, which we probably don't want.